wikidatarefisland/pipes/pipe_one.py

```diff
@@ -0,0 +1,47 @@
+from itertools import chain
+from wikidatarefisland.pipes import statement_filters
+from .abstract_pipe import AbstractPipe
+from wikidatarefisland.data_model import statement_filterer
+
+
+class ItemExtractorPipe(AbstractPipe):
+    def __init__(self, external_id_formatter, storage,
+                 blacklisted_properties=None):
+        self._storage = storage
+        if blacklisted_properties is None:
+            blacklisted_properties = []
+        self.blacklisted_properties = blacklisted_properties
+        self.external_id_formatter = external_id_formatter
+
+    def flow(self, input_data):
+        return self._process_item(input_data)
+
+    def _process_item(self, item_data):
+        all_statements = chain.from_iterable([i for i in item_data['claims'].values()])
+        statement_filterers = [
+            statement_filters.referenced_statement_excluder,
+            statement_filters.external_id_statement_excluder,
+            statement_filters.get_property_id_statement_excluder(self.blacklisted_properties)
+        ]
+        potentially_ref_statement_filterer = statement_filterer.StatementFilterer(
```

Comment:
This is a little confusing, imho we should explicitly import the class

Reply:
Agreed